### ISSUE TITLE
skia-python install error.

### ISSUE THREAD

**Issue by exdj21** [Tremeschin/modular-music-visualizer#3]
Hi,

 I'm trying to install the requirements on windows but it's giving an error:

ERROR: Could not find a version that satisfies the requirement skia-python==85.0 (from versions: none)
ERROR: No matching distribution found for skia-python==85.0

or even direct with: "pip install skia-python"

specs:
Windows 10 64bits
python 3.7.2
pip 20.2.2
virtualenv 20.0.21

**Comment by Tremeschin** [comment]
Yup I tried installing skia-python on Windows and no success for (I think it took) about two hours.

They have instructions on that [here](https://kyamagu.github.io/skia-python/install.html#windows) but some build files just fails, it was a weird experience and there were pathing issues on some files I had to manually change.

I got up to the last step before compiling skia and as mentioned the build fails.

Perhaps going to the official skia website and see their instructions there? (it's pretty similar to this skia-python docs)..

Would be good if someone could upload a Windows skia-python package to pypi for it to be available on the OS.

And also I don't really have an proper Windows test environment for now, I might be able to test under a VM perhaps and not over controlling friends pc on a distance lol

I switched rendering backends a lot throughout MMV development, it used to be pure numpy and PIL (100% would work on windows) until I got up to vector graphics for the bars, had to use Cairo for it and I think it is harder to install than skia there, however skia was just faster and could render stuff on the GPU with many cool effects like drop shadows, there are some shaders I can set as well.

But if you really wanna use it, we can also do the opposite: you running a Linux environment for now either on a VM or on Microsoft's WSL, this last one could work very well ngl, whenever I have a proper Windows environment I'll experiment with it.

Feel free to ask for more instructions or explanations :)

TL;DR: README says it, MMV is (officially (tm)) Linux only now, need help on installing skia-python on Windows

**Comment by exdj21** [comment]
Now I tried to install* it on Raspberry Pi 3, running Raspbian.

same error:

`Collecting skia-python==85.0 (from -r mmv/requirements.txt (line 33))`
`Could not find a version that satisfies the requirement skia-python==85.0 (from -r mmv/requirements.txt (line 33)) (from versions: )`
`No matching distribution found for skia-python==85.0 (from -r mmv/requirements.txt (line 33))`
`(venv) pi@raspberrypi:~/modular-music-visualizer/mmv $ `

*following your install steps.

**Comment by exdj21** [comment]
> Now I tried to install* it on Raspberry Pi 3, running Raspbian.
> 
> same error:
> 
> `Collecting skia-python==85.0 (from -r mmv/requirements.txt (line 33))`
> `Could not find a version that satisfies the requirement skia-python==85.0 (from -r mmv/requirements.txt (line 33)) (from versions: )`
> `No matching distribution found for skia-python==85.0 (from -r mmv/requirements.txt (line 33))`
> `(venv) pi@raspberrypi:~/modular-music-visualizer/mmv $ `
> 
> *following your install steps.

But I just found the problem:
*skia-python requirements =>Linux x86_64

**Comment by Tremeschin** [comment]
Remove the skia-python==version, this one might only be available for Python 3.8 I use, try only `pip install skia-python`

**Comment by Tremeschin** [comment]
Yea `skia-python` is only available for 64 bits architectures, instructions on commit 2bf77a9, updated README with that info.

I could install `skia-python` from PyPI on Wine with a 64 bit Python installation, can you test it natively on Windows and on the Raspberry Pi? I'm curious on the performance on such hardware.
